paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -749,8 +749,28 @@
                         <li><a href="#">User Agreement</a></li>
                         <li><a href="#">Privacy</a></li>
                         <li><a href="#">About us</a></li>
-                        <li><a href="#">Contact</a></li>
+                        <li><a href="#" id="preferences" class="d-flex align-items-center">
+                          <div class="mr-2">
+                            <img src="/assets/publisher/images/Union.svg" alt="">
+                          </div>
+                          <div>
+                            Marketing preferences
+                          </div>
+                        </a></li>
                     </ul>
+                  <script>
+                    document.querySelector('#preferences')
+                      .addEventListener('click', (e) => {
+                        e.preventDefault();
+                        if (PAF.getIdsAndPreferences()) {
+                          PAFUI.promptConsent();
+                        } else {
+                          document.cookie = 'paf_identifiers=;';
```

Comment:
the name of these cookies should not be hardcoded here.
Can't you just ignore these two lines? The `cmpCheck` will refresh cookies if needed, doesn't it?

Reply:
No, I cannot ignore it. CMP will get `undefined` and will skip prompt

Reply:
I see only one way to implement it. It is to provide an argument to cmpCheck to force prompt